src/app/modules/funds/funds.service.ts

```diff
@@ -27,7 +29,11 @@ export class FundsService {
       });
   };
 
+  update = (fund: Fund, id: number): Observable<Fund> => {
+    return this.http.put<Fund>(`${this.api}/update/${id}`, fund);
+  };
+
   saveFundsToSubject = (fund: Fund): void => this.subject.next({fund});
 
-  getSavedFund = (): Observable<any> => this.subject.asObservable();
+  getSavedFund = (): Observable<TFund> => this.subject.asObservable();
```

Comment:
Just a tip: you can sort these methods by targets, e.g. "get" methods first, after them "save" methods and so on.